### PR TITLE
part 2 solution added :)

### PR DESCRIPTION
**Pull request by rikpatel98**

### CODE REVIEW

File: Day 2/Day 2 Solution.ipynb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+   "source": [
+    "for i in newsteps:\n",
+    "    if i.split()[0] == 'forward':\n",
+    "        horizontal += int(i.split()[1])\n",
+    "    elif i.split()[0] == 'down':\n",
+    "        depth += int(i.split()[1])\n",
+    "    elif i.split()[0] == 'up':\n",
+    "        depth -= int(i.split()[1])\n",
+    "    else:\n",
+    "        print('nerd')"
```
Comment (jackopenn) [inline Day 2/Day 2 Solution.ipynb]: This is quite inappropriate, please change to `error`.

File: Day 2/Day 2 Solution.ipynb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
+      "\u001b[1;32m<ipython-input-30-3aedf3ee4047>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     10\u001b[0m \u001b[0mnewsteps\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;33m[\u001b[0m\u001b[0mi\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0mi\u001b[0m \u001b[1;32min\u001b[0m \u001b[0msteps\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     11\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 12\u001b[1;33m \u001b[0mnewsteps\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mtype\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     13\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
+      "\u001b[1;31mAttributeError\u001b[0m: 'str' object has no attribute 'type'"
```
Comment (jackopenn) [inline Day 2/Day 2 Solution.ipynb]: I don't want to see this error in the repo, please hide.

File: Day 2/Day 2 Solution.ipynb [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      "text/plain": [
+       "4962"
+      ]
+     },
+     "execution_count": 64,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": []
```
Comment (jackopenn) [inline Day 2/Day 2 Solution.ipynb]: Remove these empty cells.

**Review by jackopenn — CHANGES_REQUESTED** [review]
Please fix the changes i've requested.

File: Day 2/Day 2 Solution.ipynb (hunk 1 above)
Comment (rikpatel98) [reply Day 2/Day 2 Solution.ipynb]: changes made :)

File: Day 2/Day 2 Solution.ipynb (hunk 2 above)
Comment (rikpatel98) [reply Day 2/Day 2 Solution.ipynb]: changes made :)

File: Day 2/Day 2 Solution.ipynb (hunk 3 above)
Comment (rikpatel98) [reply Day 2/Day 2 Solution.ipynb]: changes made :)

**Comment by jackopenn** [comment]
LGTM, part 2 please @rikpatel98

**Review by jackopenn — APPROVED** [review]
LGTM :)

**Pull request merged**